lib/cylc/batch_sys_handlers/pbs.py

```diff
@@ -39,6 +39,7 @@ class PBSHandler(object):
     # N.B. The "qstat JOB_ID" command returns 1 if JOB_ID is no longer in the
     # system, so there is no need to filter its output.
     POLL_CMD = "qstat"
+    POLL_CANT_CONNECT_ERR = "Connection refused"
```

Comment:
How stable is this message across different `pbs` versions?

Reply:
We only have our own to test on our site, so no idea. Our admin team is confident that it is a good message to use though. @hjoliver may have a better insight with his connection with the people who develop PBS.

Reply:
I'll ping my PBS connections...

Reply:
(however, we can always adapt to other PBS versions in future releases if need be)

Reply:
(... pinged)